### ISSUE TITLE
[Bug] Generate video with Hardcoded sub crash

### ISSUE THREAD

**Issue by OmrSi** [SubtitleEdit/subtitleedit#6170]
My friend was trying to use a 35GB video to generate a video with subtitles but he got this error:

![image](https://user-images.githubusercontent.com/20923700/185941551-28dde6ed-0507-4a97-a875-290fe98139db.png)

I would look into it myself but I don't have a video larger than 4GB. ^^"

EDIT: I guess it's as simple as editing the max value of the `numericUpDownTargetFileSize`, but the question is what the appropriate max value here? How large can a video file be?

**Comment by OmrSi** [comment]
Video info:

```
Video
ID                                       : 1
ID in the original source medium         : 4113 (0x1011)
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4.1
Format settings                          : CABAC / 2 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 2 frames
Codec ID                                 : V_MPEG4/ISO/AVC
Duration                                 : 2 h 17 min
Bit rate mode                            : Variable
Bit rate                                 : 26.9 Mb/s
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.541
Stream size                              : 25.8 GiB (75%)
Language                                 : English
Default                                  : No
Forced                                   : No
Original source medium                   : Blu-ray
```

**Comment by OmrSi** [comment]
@niksedk If you could tell me what value you see appropriate, I can edit the maximum value myself and make a PR.

**Comment by niksedk** [comment]
Ah, yes. I've made this error before... I think the solution is to set the progress bar max to 100 - and then calculate the percentage during progress.

**Comment by OmrSi** [comment]
@niksedk I'm not sure if this is related to the progress bar or the `numericUpDownTargetFileSize`.

**Comment by niksedk** [comment]
> @niksedk I'm not sure if this is related to the progress bar or the `numericUpDownTargetFileSize`.

Yes, thx - updated comment ;)

**Comment by OmrSi** [comment]
Yeah, I think `1000GB` is more than enough.

**Comment by OmrSi** [comment]
Yep, this is why it was crashing. That's what I think the stack trace is showing since it's in `OnShown`.
Will you change it? I guess it's not worth a PR. ^^"

**Comment by niksedk** [comment]
OK, should be fixed in commit e9321c2c131ffa2cd4dde20ca9b1485569e95d0e

**Comment by OmrSi** [comment]
Great! Thank you.
